DataConnectors/Box/Box_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "BoxDataConnector",
+    "title": "Box (Preview)",
+    "publisher": "Box",
+    "descriptionMarkdown": "The Box data connector provides the capability to ingest [Box enterprise's events](https://developer.box.com/guides/events/#admin-events) into Azure Sentinel using the Box REST API. Refer to [Box  documentation](https://developer.box.com/guides/events/for-enterprise/) for more information.",
+    "additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. Follow the steps to use this Kusto functions alias **BoxEvents** in queries and workbooks. [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-BoxDataConnector-parser) ",
+    "graphQueries": [
+        {
+            "metricName": "Box events",
+            "legend": "Box events",
+            "baseQuery": "BoxEvents_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All Box events",
+            "query": "BoxEvents\n| sort by TimeGenerated desc"
+        }
+     ],
+    "dataTypes": [
+        {
+            "name": "BoxEvents_CL",
+            "lastDataReceivedQuery": "BoxEvents_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "BoxEvents_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(1d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
```

Comment:
Could you please update the permissiondisplaytext to "read and write permissions on the workspace are required.".

Thanks,
Aditi Maheshwari